apps/roboshield/src/components/RichText/styleConverter.tsx

```diff
@@ -0,0 +1,48 @@
+import { JSXConverters } from "@payloadcms/richtext-lexical/react";
+import { RichTypography, Link } from "@commons-ui/next";
+
+const DEFAULT_PROPS = {
+  html: false,
+};
+
+export const styleConverter = (converterProps: any): JSXConverters<any> => ({
+  heading: ({ node, nodesToJSX }) => {
+    const Tag = node.tag || "h1";
+    return (
+      <RichTypography
+        variant={Tag}
+        component={Tag}
+        {...converterProps.typographyProps}
+      >
+        {nodesToJSX({ nodes: node.children })}
+      </RichTypography>
+    );
+  },
+  quote: ({ node, nodesToJSX }) => (
+    <blockquote>{nodesToJSX({ nodes: node.children })}</blockquote>
+  ),
+  link: ({ node, nodesToJSX }) => {
+    return (
+      <RichTypography
+        component={Link}
+        href={node.fields.url || node.href || ""}
```

Comment:
There is no standard prop for href?

Reply:
@kilemensi I did a couple of tests and realized we actually don't need this.  I struggled to parse internal links and its best we don't add a case to intercept how we format them i.e. the default behaviour of the default Convertor works consistently with what we have on prod

Reply:
Then move it to an issue we can track separately @m453h (as we may need to be able to style links separately but it's not needed _today_).